tests/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+
+services:
+  dev-pihole:
+    image: ghcr.io/pi-hole/pihole:latest
+    volumes:
+      - ../:/var/www/html/admin:ro
```

Comment:
To have functional frontend tests, you need a backend too. I achieved this by using the official docker image and overlaying the repo code.